packages/pulpcore-selinux/pulpcore-selinux.spec

```diff
@@ -60,7 +61,7 @@ do
   done
 done
 systemctl daemon-reexec &>/dev/null || :
-/sbin/fixfiles -R python3-pulpcore restore || :
+/sbin/fixfiles -R python%{pulpcore_python}-pulpcore restore || :
```

Comment:
there is a BZ about this somewhere

Reply:
Thanks, let me update that BZ with the right version